lib/constants.js

```diff
@@ -77,3 +77,6 @@ export const ITEM_ALLOW_EDITS = [
   // FAQ, privacy policy, changelog, content guidelines
   349, 76894, 78763, 81862
 ]
+
+export const INVOICE_RETENTION_DAYS = 7
+export const FORGOTTEN_HASH = 'f0rg0tten'
```

Comment:
I would use 32 zero bytes since that's a common practice for "unused data", I think:

```python-repl
$ python
Python 3.11.5 (main, Sep  2 2023, 14:16:33) [GCC 13.2.1 20230801] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> bytes(32).hex()
'0000000000000000000000000000000000000000000000000000000000000000'
```

But there is another problem that I also mentioned in another comment: There is a unique index on hash and I think we should keep this unique index. Is deleting the hash even relevant for privacy? Afaik, it's just random bytes so there is no PII in there :thinking:

Reply:
Allowing null per my other comment should still let us keep the unique key. (I think the security issue has to do not only with the randomness of the hash but also with who else might have a copy of it.)